server/src/Utopia/Web/Github.hs

```diff
@@ -294,6 +294,17 @@ listPullRequests accessToken owner repository page possibleHead = do
   let queryParameters = [("page", show page)] <> headParameter
   callGithub getFromGithub queryParameters listPullRequestsErrorCases accessToken repoUrl ()
 
+getGithubUserErrorCases :: (MonadIO m) => Status -> ExceptT Text m a
+getGithubUserErrorCases status | status == notModified304                   = liftIO $ fail "Not modified returned for Github user."
```

Comment:
Why is the 304 an explicitly handled failure case?

Reply:
I just handle the cases like this that Github says they might return.